src/overlays/actors/ovl_Bg_Jya_Goroiwa/z_bg_jya_goroiwa.c

```diff
@@ -8,8 +14,14 @@ void BgJyaGoroiwa_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaGoroiwa_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaGoroiwa_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaGoroiwa_Draw(Actor* thisx, GlobalContext* globalCtx);
+void func_80897DF0(BgJyaGoroiwa* this, GlobalContext* globalCtx);
```

Comment:
Also it's slightly better to separate generic actor functions from internal ones.
```suggestion
void BgJyaGoroiwa_Draw(Actor* thisx, GlobalContext* globalCtx);

void func_80897DF0(BgJyaGoroiwa* this, GlobalContext* globalCtx);
```